bbc_image_scraper/bbc_image_scraper/spiders/bbc_spider.py

```diff
@@ -5,7 +5,7 @@
 class BbcImagesSpider(scrapy.Spider):
```

Comment:
This should be named `BBCSpider` instead of `BbcImagesSpider` -- I think that would make more sense.